### PR TITLE
zh-translation: embed/go/tensorflow.md

### PR DESCRIPTION
**Pull request by majinghe**
tranlsation `mbed/go/tensorflow.md` file and corresponding issue is #1093 .

### CODE REVIEW

File: docs/book/zh/src/embed/go/tensorflow.md [hunk 1]
```diff
@@ -1 +1,122 @@
 # Tensorflow
+
+在这一章节，我们将向你展示如何在 Rust 中创建一个用于图像分类的 Tensorflow 推理函数，然后将其嵌入到 Go 应用程序中。项目的源码可以在[这儿](https://github.com/second-state/WasmEdge-go-examples/tree/master/wasmedge-bindgen/go_TfliteFood)查看。
+
+## Rust function compiled into WebAssembly
```
Comment (alabulei1) [inline docs/book/zh/src/embed/go/tensorflow.md:5]: 标题没翻译哈

File: docs/book/zh/src/embed/go/tensorflow.md [hunk 2]
```diff
@@ -1 +1,122 @@
...
+
+	bg.Release()
+	vm.Release()
+	conf.Release()
+	tfobj.Release()
+	tfliteobj.Release()
+}
+```
+
+## Build and run
```
Comment (alabulei1) [inline docs/book/zh/src/embed/go/tensorflow.md:99]: 英文忘记删了

File: docs/book/zh/src/embed/go/tensorflow.md [hunk 3]
```diff
@@ -1 +1,122 @@
...
+
+你也可以使用我们的 AOT 编译器 `wasmedgec` 来检测 WebAssembly 文件，以便让其运行的更快。[学习更多](../../start/universal.md)。
+
+```bash
+$ wasmedgec rust_tflite_food_lib.wasm rust_tflite_food_lib.wasm
+```
+
+## Go host 应用程序
+
+[Go host 应用程序](https://github.com/second-state/WasmEdge-go-examples/blob/master/wasmedge-bindgen/go_TfliteFood/tflite_food.go)的源码展示了如何用 Tensorflow 来扩展并实例化一个 WasmEdge 运行时，以及如何将图片数据传递给 WasmEdge 中的 Rust 函数，以便来执行推理。
```
Comment (alabulei1) [inline docs/book/zh/src/embed/go/tensorflow.md:48]: WasmEdge Runtime 连在一起，可以不翻译 runtime 哈

**Review by alabulei1 — COMMENTED** [review]
Great! Thank you for your contribution.

**Comment by majinghe** [comment]
@alabulei1  fix the translation typo, PTAL.

**Comment by alabulei1** [comment]
LGTM! 
@hydai Could you help merge this PR?

**Review by hydai — APPROVED** [review]

**Pull request merged**